src/status_im/data_store/realm/core.cljs

```diff
@@ -285,16 +285,25 @@
 
 ;; realm functions
 
+(defn object-stored? [obj]
+  (and (not (nil? obj)) (.isValid obj)))
+
 (defn write [realm f]
   (.write realm f))
 
 (defn create
   ([realm schema-name obj]
    (create realm schema-name obj false))
   ([realm schema-name obj update?]
+   (create realm schema-name obj update? nil))
+  ([realm schema-name obj update? on-success]
    (let [obj-to-save (select-keys obj (keys (get-in entity->schemas
-                                                    [schema-name :properties])))]
-     (.create realm (name schema-name) (clj->js obj-to-save) update?))))
+                                                    [schema-name :properties])))
+         obj-in-realm (.create realm (name schema-name) (clj->js obj-to-save) update?)]
+     (when (and (not (nil? on-success))
```

Comment:
`(not (nil?` can be replaced with `fn?` here

Reply:
oh, thanks! I didn't know about this one...